Fix formatting issue for diagnostic messages

praveenkuttappan opened the pull request:
Changes to show border for diag messages and to fix padding issue. Previous change was setting background color at each <p> node that has messed the layout. I have updated this to group all diagnostic messages of same type and show it as row.

Reverted background color for error and updated warning color to lightyellow.

praveenkuttappan commented:
Snip from test run for  single line diagnostic.

![image](https://user-images.githubusercontent.com/55455725/93474421-2cecc800-f8ac-11ea-8f74-28c9c7dc9036.png)

praveenkuttappan commented:
Snip of multi line diagnostics.
![image](https://user-images.githubusercontent.com/55455725/93474581-6291b100-f8ac-11ea-816d-687cbe0c184d.png)

praveenkuttappan commented:
Snip of multi line with multiple levels from a test run. Please ignore dummy lines.
![image](https://user-images.githubusercontent.com/55455725/93474684-8bb24180-f8ac-11ea-882b-2dcc1c18e992.png)

pakrym commented on src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:
```diff
@@ -23,33 +23,17 @@
 </tr>
 @if (Model.Diagnostics.Any())
 {
+    var errorDiags = Model.Diagnostics.Where(d => d.Level == 0 || d.Level == APIView.CodeDiagnosticLevel.Error);
```
Add a Default member to CodeDiagnosticLevel with value 0

praveenkuttappan replied:
Done

pakrym commented:
Can we make the link less bright/bold as well?

praveenkuttappan commented:
I have removed bold font added for the details.

pakrym commented:
Can you share a screenshot, please?

praveenkuttappan commented:
Screenshot with updated font for details link.
![image](https://user-images.githubusercontent.com/55455725/93502831-20786780-f8cc-11ea-82ae-8193a8e37f9c.png)

praveenkuttappan commented:
No change since last commit for multiple diag types. Screenshot from latest test run.
![image](https://user-images.githubusercontent.com/55455725/93502953-4bfb5200-f8cc-11ea-998e-9c9d6a106f6e.png)
